src/lib/falcor/FalcorController.js

```diff
@@ -34,41 +34,76 @@ export default class FalcorController extends BaseComponent {
   // this should always contain all the data the component needs
   loadFalcorCache() {
```

Comment:
I think this should take falcorPathSets as the argument rather than generating it itself.